Britannia/common/on_action/BTWK_death.txt

```diff
@@ -0,0 +1,27 @@
+﻿on_death = {
+	on_actions = {BTWK_uther_dies_on_action}
+}
+
+
+#Trigger to return Arthur to Dumnonia a few weeks after Uther's death (simulate time for news to reach Arthur, & for his return)
+
+BTWK_uther_dies_on_action = {
+	effect = {
+		if = {
+			limit = {
+				trigger_if = {
```

Comment:
Do you actually need a trigger_if here? You're already in a limit block...

Reply:
Just remnant code from when I was trying to fix it, will sort it out